internal/index/manager/manager.go

```diff
@@ -721,12 +777,15 @@ func (mgr *Manager) updateTagJob(name string, t tag, tagDetails map[string]query
 	}
 }
 
-func (mgr *Manager) ImportPcap(filename string) {
+func (mgr *Manager) ImportPcaps(filenames []string) {
+	if len(filenames) == 0 {
+		return
+	}
 	mgr.jobs <- func() {
 		//add job to be processed by importer goroutine
-		mgr.importJobs = append(mgr.importJobs, filename)
+		mgr.importJobs = append(mgr.importJobs, filenames...)
 		//start import job when none running
-		if len(mgr.importJobs) == 1 {
+		if len(mgr.importJobs) == len(filenames) {
 			indexes, releaser := mgr.getIndexesCopy(0)
 			go mgr.importPcapJob(mgr.importJobs[:1], mgr.nextStreamID, indexes, releaser)
```

Comment:
I think we can import all pcaps at once here instead of only the first one.

Reply:
yes, fixed!